shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapterTest.java

```diff
@@ -123,7 +157,7 @@ public void dropPartitionEvent() {
 
     MetaStoreEvent expected = MetaStoreEvent
         .builder(EventType.ON_DROP_PARTITION, TEST_DB, TEST_TABLE)
-        .partitionColumns(PARTITION_COLUMNS)
+        .partitionColumns(new ArrayList<String>(PARTITION_KEYS_MAP.keySet()))
```

Comment:
is the `keySet()` not going to give you a different order in different environments?

Reply:
It will maintain the order provided by the user as the map is backed by an Array and the iteration is performed using Set which is also backed by an Array.

Documentation says that the iteration is deterministic. https://google.github.io/guava/releases/19.0/api/docs/com/google/common/collect/ImmutableCollection.html